setup.py

```diff
@@ -31,6 +31,7 @@
         "jupyter",
         "jupytext",
         "seaborn",
+        "xlrd",
```

Comment:
Why do we need to import this library? `openpyxl` doesn't work for loading Polish bonds xls?

Reply:
It only supports XLSX, not XLS :(